src/oc_erchef/apps/oc_chef_wm/src/chef_reindex.erl

```diff
@@ -255,8 +256,14 @@ log_failures(OrgName, [Failure|Rest]) ->
 humanize_failures([], Acc) ->
     Acc;
 humanize_failures([H|T], Acc) ->
-    {Id, Reason} = H,
-    humanize_failures(T, [{Id, pretty_reason(Reason)} | Acc]).
+    case H of
+        {Id, Reason} ->
+            humanize_failures(T, [{Id, pretty_reason(Reason)} | Acc]);
+        X ->
+            lager:error("humanize failures couldn't make sense of failure ~s~p", [X]),
+            humanize_failures(T, [{Id, pretty_reason(Reason)} | Acc]);
```

Comment:
Id and Reason are unbound here, maybe just make X our reason and put some dummy ID in